### ISSUE TITLE
Data augmentation for SVHN dataset

### ISSUE THREAD

**Issue by QiushiYang** [google-research/fixmatch#71]
Many thanks for your clean codes. 

(1) It seems there is no code of dataloader for SVHN dataset. Would you like to introduce what kinds of data pre-processing and augmenation methods (and mean&std values used for normalization) you used for SVHN dataset in the training phase?

(2) Also, as for the results reported in Table 2 of your paper, I wonder which version of SVHN dataset (it has two versions, one including 73257 training images and 26032 test images, the other one named extra version containing 631131 images) you used to train FixMatch models?

**Comment by carlini** [comment]
The SVHN dataloader can be found here:

https://github.com/google-research/fixmatch/blob/d4985a158065947dba803e626ee9a6721709c570/libml/data.py#L297-L300

We run experiments with the smaller svhn_noextra dataset here.

**Comment by QiushiYang** [comment]
Thanks a lot!
Do you mean that you used smaller version of SVHN dataset to train and to test, without the svhn_extra?

**Comment by carlini** [comment]
Yeah, we used the smaller dataset to train. This dataset, in the code, is called `svhn_noextra`. If you look at the ReMixMatch paper I think we also have results for how much gain you et by moving to the full svhn dataset, and it was rather small.

**Comment by QiushiYang** [comment]
Many thanks! I will have a try :-)
